From eddc38e71f15deaf045abcde90b04016dcf7c97d Mon Sep 17 00:00:00 2001
From: Peer Adelt <peer.adelt@hni.uni-paderborn.de>
Date: Mon, 26 Oct 2020 04:08:00 +0100
Subject: [PATCH] plugin: Fixes compiling errors on OSX

Removed the following entries
 - "qemu_plugin_register_vcpu_mem_haddr_cb"
 - "qemu_plugin_ram_addr_from_host"
 - "qemu_plugin_hwaddr_to_raddr"
from file "plugins/qemu-plugins.symbols".

The exported symbols do not seem to be defined anywhere and cause
errors when linking under OSX Mojave and Big Sur.

Signed-off-by: Peer Adelt <peer.adelt@hni.uni-paderborn.de>
---
 plugins/qemu-plugins.symbols | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/plugins/qemu-plugins.symbols b/plugins/qemu-plugins.symbols
index 4bdb381f48..40b4ff3821 100644
--- a/plugins/qemu-plugins.symbols
+++ b/plugins/qemu-plugins.symbols
@@ -8,9 +8,7 @@
   qemu_plugin_register_vcpu_insn_exec_cb;
   qemu_plugin_register_vcpu_insn_exec_inline;
   qemu_plugin_register_vcpu_mem_cb;
-  qemu_plugin_register_vcpu_mem_haddr_cb;
   qemu_plugin_register_vcpu_mem_inline;
-  qemu_plugin_ram_addr_from_host;
   qemu_plugin_register_vcpu_tb_trans_cb;
   qemu_plugin_register_vcpu_tb_exec_cb;
   qemu_plugin_register_vcpu_tb_exec_inline;
@@ -32,7 +30,6 @@
   qemu_plugin_mem_is_store;
   qemu_plugin_get_hwaddr;
   qemu_plugin_hwaddr_is_io;
-  qemu_plugin_hwaddr_to_raddr;
   qemu_plugin_vcpu_for_each;
   qemu_plugin_n_vcpus;
   qemu_plugin_n_max_vcpus;
-- 
2.29.1

